contrib/README.md

```diff
@@ -22,5 +22,5 @@ Vault scripts provided here use this facility to retrieve the vault secret from
 contributions welcome
 =====================
 
-Send in pull requests to add scripts of your own.  The sky is the limit!
+Send in pull requests to fix bugs in the scripts and add inventory plugins of your own. New features should be added to inventory plugins with accompanying tests.
```

Comment:
this still seems to invite features and new scripts (most people will not realize plugins are different).

`We are only accepting bugfixes for inventory scripts, if you want new ones or to add features target inventory plugins instead <link>.`

Reply:
That's clearer, thanks.